### DISCUSSION TITLE
BBackupp v2.0 Feature Request

### DISCUSSION THREAD

**Discussion by Lakr233** [Lakr233/BBackupp#9] [Ideas]
For any feature to be done in v2.0, put it here.

v3.0 will switch to Linux and Docker, so 2.0 will be the last version that comes with Mac app.

**Comment by Innei** [comment]
支持备份中断恢复

**Reply by Lakr233** [reply]
iOS 限制 做不了

**Comment by YukiHakarigoto** [comment]
### 单独备份相册照片
背景：
1. 隐私安全（云 上 贵 州）~~iCloud 订阅太贵了~~
2. 父母老人手机里就照片比较重要。
3. 存在 Nas 管理和共享方便些。

实现：
1. 进入家里局域网后，自动备份设备中的新照片。
2. 照片可以选择存储在 Nas 的指定位置。

**Reply by Lakr233** [reply]
相册可以单独读取 这个做到备份 profile 吧

**Comment by ImSingee** [comment]
希望可以在笔记本上实现合盖备份

拆分下来其实是两个

1. 备份期间不睡眠，这个比较简单，有现成的脚本来实现
2. 预定的时间自动唤醒备份（这个感觉不太容易实现）

**Comment by ImSingee** [comment]
希望可以支持自动将备份「转移」到 NAS，即备份的时候在本地硬盘，备份完成后再移动到 NAS（考虑到增量备份，可以保留最近的一个备份在本地），这样要比直接备份目的地选择 NAS 要更稳定、速度也更快

**Comment by shunichio** [comment]
哈哈，想知道现在2.0进度如何了呀？

**Reply by Lakr233** [reply]
在做了在做了

**Comment by Lakr233** [comment]
> 希望可以支持自动将备份「转移」到 NAS，即备份的时候在本地硬盘，备份完成后再移动到 NAS（考虑到增量备份，可以保留最近的一个备份在本地），这样要比直接备份目的地选择 NAS 要更稳定、速度也更快

已完成！

**Reply by ImSingee** [reply]
cool👀 爱了

**Comment by Lakr233** [comment]
> 哈哈，想知道现在2.0进度如何了呀？

基本写完了

**Comment by Lakr233** [comment]
> 希望可以在笔记本上实现合盖备份
> 
> 拆分下来其实是两个
> 
> 1. 备份期间不睡眠，这个比较简单，有现成的脚本来实现
> 2. 预定的时间自动唤醒备份（这个感觉不太容易实现）

这个应该不会做
